pkgs/by-name/fc/fcitx5-material-color/package.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenvNoCC
+, fetchFromGitHub
+, lib
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "fcitx5-material-color";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "hosxy";
+    repo = "Fcitx5-Material-Color";
+    rev = "${finalAttrs.version}";
+    hash = "sha256-i9JHIJ+cHLTBZUNzj9Ujl3LIdkCllTWpO1Ta4OT1LTc=";
+  };
+
+  installPhase = ''
+    install -Dm644 arrow.png radio.png -t $out/share/${finalAttrs.pname}/
+    for _variant in black blue brown deepPurple indigo orange pink red sakuraPink teal; do
+      _variant_name=Material-Color-$_variant
+      install -dm755 $_variant_name $out/share/fcitx5/themes/$_variant_name
+      ln -sv ../../../$pname/arrow.png $out/share/fcitx5/themes/$_variant_name/
+      ln -sv ../../../$pname/radio.png $out/share/fcitx5/themes/$_variant_name/
+      install -Dm644 theme-$_variant.conf $out/share/fcitx5/themes/$_variant_name/theme.conf
+      sed -i "s/^Name=.*/Name=$_variant_name/" $out/share/fcitx5/themes/$_variant_name/theme.conf
+    done
+  '';
```

Comment:
I copied this part from [archlinux cn PKGBUILD](https://gitlab.archlinux.org/archlinux/packaging/packages/fcitx5-material-color/-/blob/main/PKGBUILD?ref_type=heads#L16), and I have tested stuff-in-one-directory but it doesn't work on my computer.

![Screenshot 2023-10-26 09-49-39](https://github.com/NixOS/nixpkgs/assets/23723294/2fc271ca-4c50-480c-a363-93559be7a470)

Reply:
I remember fighting with this package on arch linux like 2 years ago or something as well. I never managed to make it work, but I hoped it would be easier with NixOS.

Could you add a TODO checkbox at the top of the PR, so we remember to at least confirm it works once before merging?

Reply:
![Screenshot 2023-10-26 10-10-22](https://github.com/NixOS/nixpkgs/assets/23723294/53023dea-2512-4205-95ce-d6fa177b68b9)

「Stuff around into each separate directory」works on my computer, could we check the checkbox

Reply:
Sure!